dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -10,16 +10,16 @@ LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
 
 MENU_001 MENUEX
 BEGIN
-    MENUITEM "Picto&grame mari", FCIDM_SHVIEW_BIGICON
+    MENUITEM "P&ictograme mari", FCIDM_SHVIEW_BIGICON
```

Comment:
&i accelerators are very uncommon. And it contradicts IDS_VIEW_LARGE "&Dale". This one is definitely a bug.

Reply:
So according to the MS screenshot it must be "&Pictograme" and "Dal&e". Please change both accordingly.

Reply:
Yes, you are right. I did not pay attention.

![2024-03-27 00_53_11-Resource Hacker - shell32 dll mui](https://github.com/reactos/reactos/assets/113553683/53774bab-cac3-413d-9c33-f1c115c85e6f)

Reply:
Now you do use a different
    MENUITEM "Dal&e", FCIDM_SHVIEW_BIGICON
    MENUITEM "&Miniaturi", FCIDM_SHVIEW_SMALLICON
vs.
    IDS_VIEW_LARGE "Pictograme mari"
    IDS_VIEW_SMALL "Pictograme mici"

I can only hope that this is correct. I do perceive that as kind of a mismatch.
Left here just as a note for now.

Reply:
Maybe it's weird but both are correct. In xp & 2k3 in the menus and in the dialog boxes there are "Dal&e" and "&Miniaturi". In String table there are "Pictograme mari" and "Pictograme mici". In newer versions they changed the translation a little bit.